q2_assembly/quast/tests/test_quast.py

```diff
@@ -486,11 +492,90 @@ def test_evaluate_contigs_action_paired_end(self, p1, p2, p3, p4):
             "tabs": [
                 {"title": "QC report", "url": "index.html"},
                 {"title": "Contig browser", "url": "q2_icarus.html"},
+                {"title": "Krona charts", "url": "q2_krona_charts.html"}
             ],
             "samples": json.dumps(["sample1", "sample2"]),
         }
         p2.assert_called_once_with(ANY, self._tmp, context=exp_context)
 
+    def make_file_tree(self, root) -> None:
+        folder1_files = [
+            os.path.join(root, f"folder_1/file_{i}") for i in range(3)
+        ]
+        folder2_files = [
+            os.path.join(root, f"folder_2/subfolder_{j}/file_{i}")
+            for i in range(3) for j in range(3)
+        ]
+        folder3_files = [
+            os.path.join(
+                root, f"folder_3/subfolder_{j}/subsubfolder_{k}/file_{i}"
+            )
+            for i in range(3) for j in range(3) for k in range(3)
+        ]
+
+        all_files = [*folder1_files, *folder2_files, *folder3_files]
+
+        for file in all_files:
+            try:
+                open(file, 'a').close()
+            except Exception:
+                os.makedirs(file)
```

Comment:
Please just directly create all those files in the `tests/data` directory - no need for another function. They will be empty anyways so no space waste and they will simplify the code greatly.